lib/anyway/rails/loaders/yaml.rb

```diff
@@ -5,11 +5,21 @@ module Rails
     module Loaders
       class YAML < Anyway::Loaders::YAML
         def load_base_yml(*)
+          parsed_yml = super
+          return parsed_yml unless environmental?(parsed_yml)
+
           super[::Rails.env] || {}
         end
 
         private
 
+        def environmental?(parsed_yml)
+          parsed_yml.keys.any? do |key|
+            envs = ::Rails.application.config.anyway_config.known_environments.dup
+            envs.concat([::Rails.env]).include?(key)
+          end
+        end
```

Comment:
Let's avoid duping configuration and perform two checks instead:
```suggestion
        def environmental?(parsed_yml)
          # likely
          return true if parsed_yml.key?(::Rails.env)
          
          # less likely
          ::Rails.application.config.anyway_config.known_environments.any? { parsed_yml.key?(_1) }
        end
```

Reply:
TIL — `_1`